.circleci/config.yml

```
@@ -41,67 +48,76 @@ default config for android apk builds: &android_defaults
 # CACHE CONFIG
 # ==============================
 
-# brew
-save brew cache: &cache_save_brew
-  name: Saving Brew cache
-  paths:
+cache keys:
+  brew ios: &key_brew_ios cache-brew-ios-v3-{{ arch }}
+  brew android: &key_brew_android cache-brew-android-v3-{{ arch }}
+  yarn: &key_yarn cache-yarn-{{ checksum "package.json" }}-{{ arch }}
+  gradle: &key_gradle cache-gradle-{{ checksum "example/android/gradle/wrapper/gradle-wrapper.properties" }}-{{ checksum "package.json" }}-{{ arch }}
+  pods: &key_pods cache-pods-v1-{{ checksum "example/ios/Podfile" }}-{{ checksum "package.json" }}-{{ arch }}
```

We should probably check in `Podfile.lock` and use that for checksum, otherwise we can't guarantee that we'll use the cache at all.
```suggestion
  pods: &key_pods cache-pods-v1-{{ checksum "example/ios/Podfile.lock" }}-{{ arch }}
```

I realize this has always been here so maybe we can fix it in a follow up?

---

`Podfile.lock` is excluded from git, so we cannot use it. Beside, we'd have to make sure we're using the same Pod version locally and on CI. So instead, I fallback to use combination of package.json and Pod file itself